ibtk/include/ibtk/CoarsenPatchStrategySet.h

```diff
@@ -160,7 +160,7 @@ class CoarsenPatchStrategySet : public SAMRAI::xfer::CoarsenPatchStrategy<NDIM>
     /*!
      * \brief The set of SAMRAI::xfer:CoarsenPatchStrategy objects.
      */
-    std::vector<SAMRAI::xfer::CoarsenPatchStrategy<NDIM>*> d_strategy_set;
+    std::vector<std::unique_ptr<SAMRAI::xfer::CoarsenPatchStrategy<NDIM> > > d_strategy_set;
```

Comment:
We should also get rid of `d_managed` here.

Reply:
Same as above.